package.json

```diff
@@ -39,10 +39,7 @@
     "webpack": "^1.12.11"
   },
   "peerDependencies": {
-    "react": "^15.5.0",
-    "react-dom": "15.5.0"
-  },
-  "dependencies": {
-    "prop-types": "^15.5.8"
+    "react": "15.4.1",
+    "react-dom": "15.4.0"
```

Comment:
@JustinRyanH What's the reason for the dependency downgrades? Our app appears to be on ^15.6.1.

Reply:
At the time ^15.4.1.